scripts/golos-boot-sequence/golos-boot-sequence.py

```diff
@@ -222,24 +222,7 @@ def createGolosAccounts():
     for acc in golosAccounts:
         if not (args.golos_genesis and acc.inGenesis):
             createAccount('cyber', acc.name, args.public_key)
-    updateAuth('gls.issuer',  'witn.major', 'active', [args.public_key], [])
```

Comment:
Please, don't remove actions from golos-boot-sequence.py. It's better to place it under `if not args.golos_genesis:`.

Reply:
Restored in 5bdd111